docker/Dockerfile

```diff
@@ -21,25 +21,14 @@ USER root
 
 # copy packages
 COPY . /home/ops/verdi/ops/standard_product
-RUN pip install --upgrade pip
-RUN pip3 install 'pyproj==1.9.6'
-RUN pip install 'pyproj==1.9.6'
-RUN pip install --upgrade --ignore-installed 'geopandas'
-#RUN pip3 install --upgrade --ignore-installed 'geopandas'
-RUN pip install shapely
-RUN pip3 install shapely
-
-RUN pip install 'fiona==1.7.13'
-RUN pip3 install 'fiona==1.7.13'
-#RUN pip3 install 'pyproj==1.9.6'
-#RUN pip install 'pyproj==1.9.6'
-#RUN pip install 'geopandas'
-#RUN pip3 install 'geopandas'
-RUN pip3 install elasticsearch
-RUN pip install elasticsearch
 
-# as ops user
-USER ops
+RUN $HOME/verdi/bin/pip install shapely
+RUN $HOME/verdi/bin/pip install 'fiona==1.7.13'
+RUN $HOME/verdi/bin/pip install 'pyproj==1.9.6'
+RUN $HOME/verdi/bin/pip install 'geopandas==0.6.3'
+RUN $HOME/verdi/bin/pip install elasticsearch
+RUN ln -s /opt/conda/pkgs/basemap-*/share/basemap $HOME/verdi/share/basemap
+
 
```

Comment:
This change was made to fix this issue in the new hysds/isce2 base image. mpl_toolkit.Basemap could not find boundary files: https://gist.github.com/shitong01/43452cfd577c7d45d1535949641f9b04#file-01-job-standard_product-s1gunw-acq_enumerator__eos-opds-python3-20200325t023643-209415z-L443-L481
Note for future: